Sources/MapboxSearch/PublicAPI/Search Results/SearchResult.swift

```diff
@@ -48,6 +48,9 @@ public protocol SearchResult {
     /// MapKit placemark
     var placemark: MKPlacemark { get }
 
+    /// An approximate distance to the origin location, in meters.
```

Comment:
It can be useful for users to know from this comment if the distance and estimatedTime are geometrical on the map or if they were calculated on the route.